### PR TITLE
average pool 3d

### PR DESCRIPTION
**Pull request by chamathabeysinghe**

### CODE REVIEW

**Comment by Dsantra92** [comment]
Looks the handle_padding function is not implemented in ivy. Maybe you want to use _handle_padding?

**Comment by Dsantra92** [comment]
The tests are failing because there is no function `handle_padding`. You probably wanted to use `_handle_padding` :)

File: ivy/functional/frontends/torch/nn/functional/pooling_functions.py [hunk 1]
```diff
@@ -106,6 +106,44 @@ def avg_pool2d(
         data_format=data_format,
     )
 
+@to_ivy_arrays_and_back
+def avg_pool3d(
```
Comment (zaeemansari70) [inline ivy/functional/frontends/torch/nn/functional/pooling_functions.py:110]: `ceil_mode`, `count_include_pad`, `divisor_override` these are not implemented in the backend yet, the frontend function needs the complete backend implementation(superset), If the backend task is open I'd suggest work on the superset behavior or else wait for it to be implemented by someone. :slightly_smiling_face: 
In the meanwhile you are free to work on any other frontend function. :slightly_smiling_face:

**Review by zaeemansari70 — COMMENTED** [review]
Hey @chamathabeysinghe! 
Thank you so much for contributing to ivy! 🙂

Your PR looks amazing ✨!
Just requested a few changes and left comments on them which needs to be addressed! 🙂

Do let me know if you have any other questions!
Thanks!

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**